CODEOWNERS

```diff
@@ -60,6 +60,6 @@ site/docs/services/omvs/ @davidradl
 
 site/docs/types/ @mandy-chessell
 
-site/docs/tutorials/docker-tutorial /  @planetf1
-site/docs/tutorials/open-metadata-labs/ @planetf1
-site/docs/tutorials/postman-tutorial / @mandy-chessell @planetf1
+site/docs/tutorials/docker-tutorial /  @mandy-chessell
```

Comment:
when view ing the pr is is saying that lines 63 and 65 have errors. could there by a invalid character on these lines?

Reply:
There is an extra space between the file name and the last /

Reply:
Thanks for the review comments. My error somehow getting those in. Fixed